crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -350,6 +350,25 @@ enum JsValueMetaKind {
     Placeholder,
 }
 
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub enum AdditionalProperty {
```

Comment:
Not a fan of this name, it's a bit too generic

Reply:
yeah, I changed it